ci/rpm7/env/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM centos:centos7
```

Comment:
We already have a Dockerfile in this repository (in docker/Dockerfile). It would be great if we could reuse that one for the CI.

Reply:
I need to split dependencies and orion build itself in separate images. I will explain why in further comments.

Reply:
NTC